Update CircleGraph (#458)
aragon/aragon-ui: src/components/CircleGraph/README.md

```jsx
import { CircleGraph } from '@aragon/ui'

const App = () => <CircleGraph value={1 / 3} />
```

## Props

### `color`

| Type                   | Default value |
| ---------------------- | ------------- |
| `Function` or `String` | None          |

The color representing the active part in the circle. When a function is passed, it takes the interpolated `value` as a unique parameter, and is expected to return a color. Avoid doing anything slow in this function as it gets called 60 times per second.

### `label`

| Type                 | Default value |
| -------------------- | ------------- |
| `Function` or `Node` | None          |

> When a React node is passed, it gets rendered directly.

Reviewer: ```suggestion
If a React node is passed, it gets rendered directly.
```

> A function that gets called to generate the label displayed on the graph. It takes the current value as a parameter and should return an object containing the following nodes:

Reviewer: ```suggestion
If a function is passed, it gets called to generate the label displayed on the graph. It takes the current value as a parameter and should return an object containing the following nodes:
```

- `value`: the main number (defaults to `Math.round(value * 100)`).

> - `suffix`: the suffix displayed after (defaults to `%`).

Reviewer: ```suggestion
- `suffix`: the suffix to display (defaults to `%`).
```

Suffixes by definition are display afterwards :)

> - `prefix`: the optional prefix to display (defaults to `<` below 1%).

Reviewer: ```suggestion
- `prefix`: the optional prefix to display (defaults to `<` when `value` is below 1).
```

```jsx
<CircleGraph label={value => Math.round(value)} />
```

### `color`

* Type: `String`
* Default: `#21c1e7`

The border color of the circle.

#### Example:

```jsx
<CircleGraph color="#FF6969" />
```

### `width`

* Type: `Number`
* Default: `4`

> Width of the circle.

Reviewer: ```suggestion
Stroke width of the circle.
```

I'd also consider renaming this prop to `stroke` or etc. "Width" of a circle usually means diameter which is definitely not what we want to convey 😄.
